man/man8/zfs.8

```diff
@@ -40,6 +40,313 @@
 .Nm
 .Fl ?V
 .Nm
+<<<<<<< 8221bcf1e4efc40d92d1354de710048c70eb709b
+=======
```

Comment:
Recently the man pages were split by subcommand, you'll want to update `man/man8/zfs-send.8`